frontend/app/templates/components/side-nav-bar.hbs

```diff
@@ -1,35 +1,55 @@
-<div class="" id="nav-side-bar-container" {{did-insert this.setupModal this}}>
-  <div class="list-group list-group-root">
-
-    <a class="list-group-item list-group-item-action top-item bg-blue-two font-weight-bold"><strong>All Teams or something</strong></a>
-
-    {{#each this.teams as |team|}}
-      <a {{on "click" (fn this.setSelectedTeam team)}} role="button" id="side-bar-team-{{team.id}}" class="list-group-item list-group-item-action {{if (eq team this.navService.selectedTeam) "bg-blue-three" "bg-blue-one"}}">
-        {{#if team.private}}
-          <img class="icon icon-button" src="/ember/assets/images/members.svg" alt="">
-        {{else}}
-          <img class="icon icon-button" src="/ember/assets/images/encrypted_small.svg" alt="">
-        {{/if}}
-        {{team.name}}
-        {{#if (or (not (eq team this.navService.selectedTeam)) this.collapsed)}}
-          <img class="icon icon-button float-right" src="/ember/assets/images/angle-left.svg" alt="<">
-        {{else}}
-          <img class="icon icon-button float-right" src="/ember/assets/images/angle-down.svg" alt="v">
-        {{/if}}
-      </a>
-      <BsCollapse class="list-group" id="nav-folders-{{team.id}}" @collapsed={{or (not (eq team this.navService.selectedTeam)) this.collapsed}}>
-        {{#each team.folders as |folder|}}
-          <a role="button"  class="list-group-item {{if (eq folder this.navService.selectedFolder) "bg-blue-two selected-nav-entry" "bg-blue-three"}}" {{on "click" (fn this.setSelectedFolder folder)}}>
-            <img class="icon icon-button" src="/ember/assets/images/folder.svg" alt="">
-            {{folder.name}}
+<div id="sidebar">
+  <nav class="navbar navbar-dark navbar-inverse">
+    <div class="navbar-header">
+      <LinkTo @route="index">
+        <img src="ember/assets/images/cryptopussy_without_text.svg" alt="logo" class="logo-icon">
+        <span class="navbar-brand">Cryptopus</span>
+      </LinkTo>
+    </div>
+    <div class="navbar navbar-collapse nav-side-bar" id="nav-side-bar-container" {{did-insert this.setupModal this}}>
```

Comment:
I am to blame, I know. However, `{{did-insert this.setupModal this}}` can be dropped?
Together with the function in `side-nav-bar.js`?